y/prices/dex/balancer/v2.py

```diff
@@ -140,7 +140,7 @@ def get_tvl(self, block: Optional[Block] = None) -> UsdValue:
     #yLazyLogger(logger)
     async def get_tvl_async(self, block: Optional[Block] = None) -> Awaitable[UsdValue]:
         balances = await self.get_balances_async(block=block)
-        return UsdValue(sum(await gather([balance.value_usd_async for balance in balances.values()])))
+        return UsdValue(sum(await gather([balance.value_usd_async for balance in balances.values() if balance.token.address != self.address])))
```

Comment:
just add a note here for future devs and we're good to go

Reply:
added in 8478d821fd4194c2f025d971244ae06ffb673085